crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -716,4 +779,44 @@ mod tests {
             "struct_namespace"
         );
     }
+
+    #[test]
+    fn test_parse_inherited_methods() {
+        let mut cxxqtdata = create_parsed_cxx_qt_data();
+
+        let unsafe_block: Item = tokens_to_syn(quote! {
+            #[cxx_qt::inherit]
+            unsafe extern "C++" {
+                fn test(self: &qobject::MyObject);
+
+                fn with_args(self: &qobject::MyObject, arg: i32);
+            }
+        });
+        let safe_block: Item = tokens_to_syn(quote! {
+            #[cxx_qt::inherit]
+            extern "C++" {
+                #[cxx_name="withRename"]
```

Comment:
wonder if we need to protect against having a `#[namespace = "N"]` here, or does this already fail ?

As inherit doesn't support a namespace as they come from the qobject instead.

Reply:
`ParsedInheritedMethod::parse_safe` already checks this.
Only the `cxx_name` attribute is allowed, all others result in a compiler error.